### PR TITLE
Integration between fitter and plotter

### LINKED ISSUE #23: Integrate fitter with plotter

### PR DESCRIPTION
**Pull request by antoniomf97**
Added integration between fitter and plotter.
This PR closes #23.

### CODE REVIEW

File: services/fitter/fitter.py [hunk 1]
```diff
@@ -11,28 +11,29 @@
 
 sys.path.append(".")
 from services.plotter.models import Configurations
+from services.plotter.plotter import graph_builder, get_data
 
 
-def get_data(file_path):
-    df = pd.read_csv(file_path, sep=",", index_col=0)
-    return df
+# def get_data(file_path):
```
Comment (easyg24) [inline services/fitter/fitter.py]: Instead of commenting, just remove please.

File: services/plotter/plotter.py [hunk 2]
```diff
@@ -19,25 +19,28 @@ def frame_builder(configs):
...
+
+def graph_builder(file, configs, funcs):
     fig = plt.figure()
 
     if not configs:
         configs = Configurations()
 
     if file:
-        data = get_data(file.file)
+        data = get_data(file)#.file)
```
Comment (easyg24) [inline services/plotter/plotter.py:39]: We need the integration with gateway and a data model.

**Review by easyg24 — CHANGES_REQUESTED** [review]
Please delete commented sections.

File: services/fitter/fitter.py (hunk 1 above)
Comment (antoniomf97) [reply services/fitter/fitter.py]: Done

**Review by easyg24 — APPROVED** [review]
Approved. But we need a new data model soon!

**Pull request merged**